perfmetrics/scripts/continuous_test/gcp_ubuntu/create_custom_vm.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import sys
+import subprocess
+
+DEFAULT_VM_NAME = "perf-tests-vm"
+DEFAULT_MACHINE_TYPE = "n2-standard-16"
+DEFAULT_IMAGE_FAMILY = "ubuntu-2004-lts"
+DEFAULT_IMAGE_PROJECT = "ubuntu-os-cloud"
+DEFAULT_BOOT_DISK_SIZE = "50GiB"
+DEFAULT_ZONE = "us-west1-a"
+DEFAULT_STARTUP_SCRIPT = "gs://anushkadhn-onb-bucket/custom_vm_startup_script.sh"
```

Comment:
You can keep it for testing only. Otherwise we have to use `--metadata-from-file=startup-script=FILE_PATH `